l10n_nl_xaf_auditfile_export/models/xaf_auditfile_export.py

```diff
@@ -96,6 +90,7 @@ def check_periods(self):
     @api.multi
     def button_generate(self):
         self.date_generated = fields.Datetime.now(self)
+        self._get_data()
```

Comment:
It seems odd that you store the data on the Model object. Why don't you return the data structures in the call to `_get_data()` and include the data structures in the `values` dictionary in the call to `render`?

Reply:
This approach works and is performant (but could create problems in inherited modules).
Your suggestion to pass the date via the values dict would solve that.
Please feel free to make a PR on top of this PR and I'll review/test the code and merge so that we can proceed with merging this functional enhancement.

Reply:
Luc, I'm reviewing your code here so I am a bit surprised that you ask me to propose a PR for a simple change like this. It seems that you think the suggestion is an improvement, but I can imagine that you think the risk is too theoretical (I believe it is two simultaneous exports on the same Odoo instance that will cause the problem) and not worth the hassle so then we can just leave it to the other reviewers to support the suggestion or not.

Reply:
stefan,
I would need to investigate if this code could fail with two simultaneous exports on the same instance but I agree that we better adapt the code. I'll put in on my todo list and try to free a bit of time before the end of next week (but I don't mind if you have time to do it earlier).

Reply:
As a committer, I can push to your branch directly. If you don't mind that, I can have a look.

Reply:
No problem, go ahead.

Reply:
Done. I also implemented @hbrunn's suggestion to have the additional template inherit from the original one. You might want to check that one.